style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+}
+
+h1 {
+  color: #172b4d;
+  font-size: 49px;
+  line-height: 52px;
+  text-align: left;
+  width: 327px;
+  padding-left: 24px;
+  padding-top: 92px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  display: flex;
+  align-items: center;
+}
+
+.prg {
+  padding-top: 12px;
+  width: 355px;
+  padding-left: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 26px;
+  color: #344563;
+  padding-bottom: 12px;
+  margin: 12px 0;
+}
+
+.nav {
+  text-align: right;
+  padding-left: 225px;
+  padding-top: 20px;
+  width: 16px;
+}
+
+.logo {
+  padding-left: 24px;
+  width: 99px;
+  height: 20px;
+  padding-top: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 19px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #6070ff;
+}
+
+.intro {
+  background-image: url(./photos/background.png);
+  background-repeat: no-repeat;
+
+  /* background-position: cover; */
```

Comment:
- [ ] Kindly avoid living commented lines in the pull request, it will help make your code cleaner